packages/components/src/Brand/Brand.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { HTMLAttributes, SVGAttributes } from "react"
-import { assetUrl } from "@kaizen/hosted-assets"
-import { OverrideClassName } from "~types/OverrideClassName"
+import { OverrideClassName } from "~components/types/OverrideClassName"
+import { assetUrl } from "~components/utils/hostedAssets"
```

Comment:
Potentially OOS: Should we deprecate the package? Or just leave it?

Reply:
Leave it, doesn't hurt anyone